src/lib/transit/routing/graph-builder.ts

```
@@ -247,21 +251,33 @@ function addCustomLocations(
   }
 
   // Connect origin to nearby stations with walking edges
+  // First find the closest station for better connectivity
+  let closestStationToOrigin = null;
+  let minDistanceToOrigin = Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)